docs/get_started/ubuntu_setup.md

```diff
@@ -64,6 +64,17 @@ You can view the installation script we just used to install MXNet for Python [h
 
 ### Install MXNet for R
 
+MXNet requires R-version to be 3.2.0 and above. If you are running earlier version of R, run below commands to update your R version, before running the installation script.
+
+```bash
+    sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys E084DAB9
```

Comment:
I suggest to use the official repo from https://cran.r-project.org/bin/linux/ubuntu/